pkg/cloudprovider/controller/nodecontroller_test.go

```diff
@@ -704,6 +729,12 @@ func TestSyncNodeStatusTransitionTime(t *testing.T) {
 									Reason:             "Node health check succeeded: kubelet /healthz endpoint returns ok",
 									LastTransitionTime: util.Date(2012, 1, 1, 0, 0, 0, 0, time.UTC),
 								},
+								{
+									Type:               api.NodeSchedulable,
+									Status:             api.ConditionFull,
```

Comment:
I meant to comment on next one...  github's review folds too much.  But looks like you also comment existing node is also schedulable.  It looks weired though, with spec not schedulable and condition scheduable